src/main/java/io/wispforest/affinity/object/AffinityBlocks.java

```diff
@@ -54,6 +55,7 @@ public class AffinityBlocks implements BlockRegistryContainer {
     @Tab(AffinityItemGroup.MAIN) public static final Block SUNSHINE_MONOLITH = new SunshineMonolithBlock();
     @Tab(AffinityItemGroup.MAIN) public static final Block ARCANE_TREETAP = new ArcaneTreetapBlock();
     @Tab(AffinityItemGroup.MAIN) public static final Block ASSEMBLY_AUGMENT = new AssemblyAugmentBlock();
+    @NoBlockItem @Tab(AffinityItemGroup.MAIN) public static final Block MANGROVE_BASKET = new MangroveBasketBlock();
```

Comment:
`BlockItemProvider` moment?

Reply:
since when is that a thing

Reply:
[![image](https://user-images.githubusercontent.com/15788224/201476302-7d072067-402d-4a2d-b312-c79246c1f5c3.png)](https://github.com/wisp-forest/affinity/commit/e271b62b0d4929624a0f79776a555801d73e7b85)